modules/acme_bucket/main.tf

```diff
@@ -10,7 +10,19 @@ resource "aws_s3_bucket" "acme_bucket" {
   }
 }
 
-resource "aws_s3_bucket" "second_acme_bucket" {
+  resource "aws_s3_bucket" "second_acme_bucket" {
+    bucket = var.bucket_name
+
+    server_side_encryption_configuration {
+      rule {
+        apply_server_side_encryption_by_default {
+          sse_algorithm = "AES256"
+        }
+      }
+    }
+  }
```

Comment:
:warning: tfsec found a **HIGH** severity issue from rule `aws-s3-no-public-buckets`:
> No public access block so not restricting public buckets

More information available [here](https://aquasecurity.github.io/tfsec/v1.26.3/checks/aws/s3/no-public-buckets/) and [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_public_access_block#restrict_public_buckets¡)